hack/test-end-to-end-docker.sh

```diff
@@ -36,7 +36,7 @@ function cleanup()
 	dump_container_logs
 
 	echo "[INFO] Dumping all resources to ${LOG_DIR}/export_all.json"
-	oc export all --all-namespaces --raw -o json --config=${ADMIN_KUBECONFIG} > ${LOG_DIR}/export_all.json
+	oc export all --all-namespaces --raw -o json --config=${ADMIN_KUBECONFIG-} > ${LOG_DIR}/export_all.json
```

Comment:
Right, I understand this, so I'm asking in what case would that be valid? In that case would we a) expect the cluster to be up already and b) if so, do we expect to get any useful information from the cluster without admin role?